src/i18n_check/utils.py

```diff
@@ -1,6 +1,256 @@
 """
-Things to include:
-    - files_to_check
+Utility functions for the i18 check keys.
 """
 
+import json
+import os 
+import re
+from pathlib import Path
+import yaml
+from collections.abc import MutableMapping
+import glob
+import string
+
+
+
+# MARK: YAML Reading
+
+# Define the path to the YAML configuration file
+config_path = Path(__file__).parent.parent.parent / ".i18n-check.yaml"
```

Comment:
There's a chance that this is going to be more robust. Specifically we need to figure out how other CLIs find their specific YAML files. The CLI should search the directory where the code is being called, odds are, but we can do this later :)